hw/ip/tlul/data/xbar_tests.hjson

```diff
@@ -0,0 +1,169 @@
+// Copyright lowRISC contributors.
```

Comment:
I think xbar_tests.hjson should live alongside xbar_sim_cfg.hjson.

Reply:
do you mean to move to tlul/generic_dv? But most of hjson files are in data/ except sim_cfg used for dvsim

Reply:
Yes. This hjson is specific to the way we run tests and regressions (whereas the others in the data dir is used for some kind of automation.

Reply:
Updated. PTAL, thanks